cms/djangoapps/contentstore/views/tests/test_course_index.py

```diff
@@ -233,7 +233,7 @@ def test_notifications_handler_dismiss(self):
             # delete nofications that are dismissed
             CourseRerunState.objects.get(id=rerun_state.id)
 
-        self.assertFalse(has_course_author_access(user2, rerun_course_key))
+        self.assertTrue(has_course_author_access(user2, rerun_course_key))
```

Comment:
Okay, traced through it again, and I agree that it seems fine.

@adampalay, @cahrens: Just giving you a heads up in case some bug crops up from this. As far as Toby and I can tell, this user *should* have course author access here as the change reflects -- not sure why they didn't before this change.